beacon-chain/p2p/service.go

```diff
@@ -295,6 +299,9 @@ func (s *Service) Stop() error {
 // Status of the p2p service. Will return an error if the service is considered unhealthy to
 // indicate that this node should not serve traffic until the issue has been resolved.
 func (s *Service) Status() error {
+	if s.isPreGenesis {
+		return nil
+	}
 	if !s.started {
 		return errors.New("not running")
 	}
```

Comment:
```suggestion
	}
	if s.startupErr != nil {
		return s.startupErr
	}
```
While you are here, pls add this